src/editor/editor.js

```diff
@@ -48,9 +48,24 @@ export default class Editor {
 	 *
 	 * Usually, not to be used directly. See the static {@link module:core/editor/editor~Editor.create `create()`} method.
 	 *
-	 * @param {Object} [config] The editor config.
+	 * @param {Object} [config={}] The editor config.
 	 */
-	constructor( config ) {
+	constructor( config = {} ) {
+		/**
+		 * @readonly
+		 * @type {module:core/context~Context}
+		 */
+		this.context = config.context || new Context();
```

Comment:
Do we need that to be public? 🤔

Reply:
🤔 looks like it doesn't have to be public.

Reply:
It is protected now.